lib/logstash/outputs/rabbitmq.rb

```diff
@@ -36,9 +36,12 @@ class RabbitMQ < LogStash::Outputs::Base
       # Should RabbitMQ persist messages to disk?
       config :persistent, :validate => :boolean, :default => true
 
+      # Additional arguments for the exchange, for example could be the presence of an alternate exchange, ...
```

Comment:
Conflict here -- the `:validate => :array` doesn't match up with your default value (a hash).

Also, the documentation for this setting isn't quite enough. Can you detail why a user would want to set this? 

Is "arguments" the right word? It feels like it is not specific enough of a word.